enclave/core/CMakeLists.txt

```diff
@@ -3,12 +3,14 @@
 
 if (UNIX)
 set(PLATFORM_SRC
-    linux/reloc.c
-    )
+	../../common/rand.S
```

Comment:
These are still using tabs instead of 4 spaces, please fix.

Reply:
done